### PR TITLE
fix: Change color of inactive bottom nav items

### LINKED ISSUE #219: [Bug] BottomNavItems look disabled
### Description

This issue fixes the display color of inactive items in the `BottomNavBar`. This issue is a spin-off from https://github.com/impierce/identity-wallet/issues/211.

### Hardware Specification

n/a

### Steps to Reproduce the Bug

Navigate to the credentials overview `/me`.

### Expected Behaviour

Inactive buttons should be rendered in a way that they do not appear to be disabled.

### Actual Behaviour

The `BottomNavBar` contains 2 `BottomNavItem`s. The active one is highlighted with a green bar, but the two inactive ones are rendered with a pale gray. This makes them look like they are disabled:

App users might assume that inactive items cannot be tapped.

### Errors

n/a

### PR DESCRIPTION
**Pull request by maiertech**
# Description of change

- This PR changes the color of inactive bottom navigation items. Refer to #219.
- I cleaned up SVG files and use `fill="currentColor"` for styling.
- I removed conditional styles, since the color is now the same for active and inactive bottom navigation items.

## Links to any relevant issues

Fixes #219.

## How the change has been tested

Manual visual test on the credentials page on the `/me` route.

## Definition of Done checklist

Add an `x` to the boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Oran-Dan — COMMENTED** [review]
I like it, but I do wonder if it might be considered inconsistent with the tabs. The unselected tabs are still greyed out, do we want those white then as well? I'm biased since i'm used to the old style so i'm not sure if new users would even notice this difference between unselected tabs on the bottom or the "All, Data, Badges" tabs, so just wondering. Curious to here what you guys think about it?

Also wondering, as I take a second look, if just lightly less white for the unselected tabs would be a bit less busy on the screen than all the same white.

**Comment by SoulaAndrikopoulos** [comment]
I agree with the points that Oran brought up. 👍 It looks nice, but we have to consider consistency with the tabs (so we follow either of the patterns; greyed out or white). If we give the tabs a white colour it would look as follows:

Screenshot 2024-05-02 at 12 27 43

At the moment, I don't have a clear preference between this and the older design, but I'd be curious to hear other's opinions. :)

**Comment by maiertech** [comment]
The question is whether a user perceives the bottom navigation as tabs. We should run this by Andrew.

**Comment by daniel-mader** [comment]
I took a look at how other popular apps style their nav bars and this is what I got for GitHub, Mastodon & Slack (all iOS). I would sum this up with: active icon is in primary color, others are _slightly_ greyed out, no indicator bar. GitHub Mastodon Slack

**Comment by daniel-mader** [comment]
If we apply the same styling to the UniMe nav bar, it would look as follows: 
light-green-none-cut
dark-green-none-cut
I would personally prefer this over the currently proposed update (this PR) which is only relying on the indicator:
Screenshot 2024-05-07 at 10 53 03
Screenshot 2024-05-07 at 10 53 21

File: unime/src/lib/components/molecules/navigation/BottomNavItem.svelte [hunk 1]
```diff
@@ -13,16 +13,12 @@
...
-      <!-- Currently, there is no better way of applying conditional Tailwind classes to a named slot inside a component. -->
+    <div class="flex h-12 w-10 flex-col items-center justify-center text-slate-800 dark:text-grey">
+      <!-- Slot accepts SVG icons, which bring their own dimensions. -->
       <slot name="icon" />
-      <div
-        class={`pt-[6px] text-[12px]/[20px] font-medium ${
-          active ? 'text-slate-800 dark:text-grey' : 'text-slate-300 dark:text-slate-500'
-        }`}
-      >
+      <div class="pt-[6px] text-[12px]/[20px] font-medium">
```
Comment (daniel-mader) [inline unime/src/lib/components/molecules/navigation/BottomNavItem.svelte:25]: Should we really drop the styling of the icon based on "active" state? (_see screenshots above_)

**Review by daniel-mader — CHANGES_REQUESTED** [review]

File: unime/src/lib/components/molecules/navigation/BottomNavItem.svelte (hunk 1 above)
Comment (maiertech) [reply unime/src/lib/components/molecules/navigation/BottomNavItem.svelte:25]: Also fine. I would, however, choose a color with AAA contrast for inactive icons.

**Comment by daniel-mader** [comment]
After a little consultation with @digital-zen, this would be the preferred outcome:

light-green-thin-2-cut

* reduce indicator thickness from 4px to 2px
* inactive items are colored in slate-800
* active item is colored in primary color
* dark theme is respected accordingly

**Review by daniel-mader — APPROVED** [review]

**Review by SoulaAndrikopoulos — APPROVED** [review]
The navbar icons are more noticeable, everything works well in the app, code looks more concise, and comments have been removed. 👍

**Pull request merged**